client/src/wikis/wiki-bar.tsx

```diff
@@ -21,21 +24,58 @@ const ArticleTitle = ({
   title: string;
   articleKey: string;
 }) => {
-  const wikiKey = useWikiStore((state) => state.wikiData.wiki.key);
+  const [wikiKey, refresh] = useWikiStore((state) => [
+    state.wikiData.wiki.key,
+    state.refresh,
+  ]);
   const { articleKey: selectedArticleKey } = useParams({ strict: false });
   const isSelected = selectedArticleKey === articleKey;
+  const navigate = useNavigate();
+  const [open, setOpen] = useState(false);
+  const wikiService = getWikiService();
 
   return (
-    <Link to="/wikis/$wikiKey/$articleKey" params={{ articleKey, wikiKey }}>
-      <p
-        className={cn(
-          isSelected && "bg-accent font-semibold",
-          "hover:bg-accent text-md w-full truncate rounded py-1 pl-6",
-        )}
-      >
-        {title}
-      </p>
-    </Link>
+    <div
+      onClick={() =>
+        navigate({
+          to: "/wikis/$wikiKey/$articleKey",
+          params: { articleKey, wikiKey },
+        })
+      }
+      className={cn(
+        isSelected && "bg-accent font-semibold",
+        "hover:bg-accent group/article flex items-center justify-between rounded",
+      )}
+    >
+      <p className="text-md w-full truncate py-1 pl-6">{title}</p>
+      <ConfirmDialog
+        title="Are you sure?"
+        description={`Do want to delete ${title} from the wiki? Deletion is definitive.`}
```

Comment:
petite proposition d'update de la desc

```suggestion
        description={<div>Do want to delete <span className="font-semibold">${title}</span> from the wiki? Deletion is definitive, and links to this article in scenes will also be deleted.</div>}
```